### PR TITLE
Add payment options section with payment fee settings

### PR DESCRIPTION
**Pull request by ademar111190**
- Add the maximum fee payment can ask
Related: https://github.com/breez/breez/pull/152

How the settings screen looks like:

|||
|---|---|
|Screenshot_20220307-104110_Breez_Debug|Screenshot_20220309-113035_Breez_Debug|

### CODE REVIEW

File: lib/l10n/app_pt.arb [hunk 1]
```diff
@@ -1465,6 +1466,12 @@
       }
     }
   },
+  "payment_options_title": "Taxas da rede relâmpago",
```
Comment (bitcoinuser) [inline lib/l10n/app_pt.arb:1469]:
```suggestion
  "payment_options_title": "Taxas da rede Relâmpago",
```

**Review by bitcoinuser — CHANGES_REQUESTED** [review]
Minor changes PT-BR

File: lib/l10n/app_fi.arb [hunk 2]
```diff
@@ -11,6 +11,7 @@
   "home_drawer_item_title_pos_settings": "Myyntipisteasetukset",
   "home_drawer_item_title_developers": "Sovelluskehittäjille",
   "home_drawer_item_title_get_refund": "Varojen lunastus",
+  "home_drawer_item_title_payment_options": "Lightning Fees",
```
Comment (tlindi) [inline lib/l10n/app_fi.arb:14]:
```suggestion
  "home_drawer_item_title_payment_options": "Salamamaksun kulut",
```

File: lib/l10n/app_fi.arb [hunk 3]
```diff
@@ -1465,6 +1466,12 @@
       }
     }
   },
+  "payment_options_title": "Lightning Fees",
+  "payment_options_fee_header": "Maximum fees for Lightning payments:",
+  "payment_options_base_fee_label": "Base Fee in sats",
+  "payment_options_proportional_fee_label": "Proportional Fee (%)",
+  "payment_options_fee_action_reset": "Reset",
+  "payment_options_fee_action_save": "Save",
```
Comment (tlindi) [inline lib/l10n/app_fi.arb:1474]:
```suggestion
  "payment_options_title": "Salamamaksun kulut",
  "payment_options_fee_header": "Salamamaksujen hyväksyttävä enimmäiskulu:",
  "payment_options_base_fee_label": "Tapahtuman perusmaksu satosheina", 
  "payment_options_proportional_fee_label": "Tapahtuman määräpohjainen kulu (%)",
  "payment_options_fee_action_reset": "Nollaa",
  "payment_options_fee_action_save": "TALLENNA",
```

**Review by tlindi — CHANGES_REQUESTED** [review]
Fix Finnish

**Review by roeierez — APPROVED** [review]

File: lib/l10n/app_pt.arb [hunk 4]
```diff
@@ -11,6 +11,7 @@
   "home_drawer_item_title_pos_settings": "Configurações do PDV",
   "home_drawer_item_title_developers": "Desenvolvedores",
   "home_drawer_item_title_get_refund": "Obter reembolso",
+  "home_drawer_item_title_payment_options": "Taxas da rede relâmpago",
```
Comment (bitcoinuser) [inline lib/l10n/app_pt.arb:14]:
```suggestion
  "home_drawer_item_title_payment_options": "Taxas de pagamento",
```

**Review by bitcoinuser — CHANGES_REQUESTED** [review]
Mudança no menu para o texto menor: Taxa de pagamento

File: lib/l10n/app_pt.arb [hunk 5]
```diff
@@ -1467,6 +1468,13 @@
   },
   "successful_payment_print": "Imprimir",
   "successful_payment_received": "Pagamento recebido!",
+  "payment_options_title": "Taxas de pagamento",
```
Comment (bitcoinuser) [inline lib/l10n/app_pt.arb:1471]:
```suggestion
  "payment_options_title": "Taxas de Pagamento",
```

**Review by bitcoinuser — CHANGES_REQUESTED** [review]
Minor change

File: lib/l10n/app_pt.arb [hunk 6]
```diff
@@ -11,6 +11,7 @@
   "home_drawer_item_title_pos_settings": "Configurações do PDV",
   "home_drawer_item_title_developers": "Desenvolvedores",
   "home_drawer_item_title_get_refund": "Obter reembolso",
+  "home_drawer_item_title_payment_options": "Taxas de pagamento",
```
Comment (bitcoinuser) [inline lib/l10n/app_pt.arb:14]:
```suggestion
  "home_drawer_item_title_payment_options": "Taxas de Pagamento",
```

**Review by bitcoinuser — CHANGES_REQUESTED** [review]
Minor change

**Comment by tlindi** [comment]
Please amends change to app_fi.arb
-  "payment_options_fee_override_enable": "Enforce maximum fees for Lightning payments",
+ "payment_options_fee_override_enable": "Määritä Salamamaksujen hyväksyttävä enimmäiskulu",

**Comment by tlindi** [comment]
app_fi.arb looks good

**Pull request merged**